app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -105,9 +129,9 @@ class MapActivity : ComponentActivity() {
             val mapView = initMapView(context, initialPosition)
 
             for (zone in getZones())
-                zone.localizations
+                zone.locations
                     .forEach {
-                        markerToLocalization[addMarkerTo(
+                        markerToLocation[addMarkerTo(
                             mapView,
                             it.position,
                             it.name,
```

Comment:
Why use a foreach here? You could simply use an inner for-loop `for (location in zone.locations) {...}`, which is more conventional and less verbose than a foreach.
Moreover, when using lambdas in kotlin, the implicit lambda argument `it` is not very expressive (even if it is convenient as it is implicitly defined), so it's usually better to rename it appropriately.
Finally, the Marker creation and its assignation in the mapping should be done in two separate lines as it is a bit heavy right now.

Reply:
Addressed